vipps-checkout-api.md

```diff
@@ -201,7 +200,7 @@ with headers
 
 The last four headers (starting with `Vipps-System-`) are meant to identify your system (and plugin). Please use self-explanatory, human readable and reasonably short values.
 
-All fields of the request body are described in our [Swagger](https://vippsas.github.io/vipps-checkout-api/#/Session/post_v2_session). The dynamic logistics callback is found by clicking the "Callback" tab in the Swagger.
+All fields of the request body are described in our [Swagger](https://vippsas.github.io/vipps-developer-docs/api/checkout#tag/Session/paths/~1v2~1session/post). The dynamic logistics callback is found by clicking the "Callback" tab in the Swagger.
```

Comment:
The last sentence of this paragraph is not relevant any more. Maybe reformulate? Or remove?

Reply:
I removed the last sentence and also changed [Swagger] to [API Reference], which is what we are saying everywhere now.